docs/bool2.md

```diff
@@ -45,6 +45,7 @@ F(A,B) = A + B
 Truth Table::
 
 |A|B|F(A,B)|
+|:-:|:-:|:-:|
```

Comment:
I was unable to see the changes done by this. the boo2.MD already has all tables with center-aligned data. Could you please help me sort this out?

Reply:
It was a mistake :sweat_smile: I just followed the pattern of how other similar tables were implemented and made the same changes here too. Funny, it was supposed to be left-aligned! :D